ibft/ibft.go

```diff
@@ -105,7 +109,7 @@ func (i *ibftImpl) StartInstance(opts StartOptions) (bool, int, []byte) {
 	go newInstance.StartMessagePipeline()
 	stageChan := newInstance.GetStageChan()
 
-	err := i.resetLeaderSelection(opts.Identifier) // Important for deterministic leader selection
+	err := i.resetLeaderSelection(i.Identifier) // Important for deterministic leader selection
```

Comment:
@nivBlox we don't need to pass a struct variable to a struct function

Reply:
the func is excepting "seed" so it seems that it should remain generic

Reply:
@nivBlox we should change resetLeaderSelection to use identifier + seq number